src/lib/components/HistoryCard.svelte

```diff
@@ -22,21 +23,29 @@ TODO:
         : history.chapter;
     $: dateFormat = formatDateAndTime(new Date(history.date));
     $: textDirection = bc.style.textDirection;
-</script>
 
-<!-- history cards are alway LTR with the reference following the text direction -->
-<div class="history-item-block dy-card w-100 bg-base-100 shadow-lg my-4" style:direction="ltr">
-    <!-- svelte-ignore a11y-click-events-have-key-events -->
-    <a
-        style="text-decoration:none;"
-        href="{base}/"
-        on:click={() =>
+    function onHistoryClick() {
+        if (history.url) {
```

Comment:
This is all good.  I hadn't even thought about the fact that the quiz should appear in history.